orbit-2-core/src/main/java/com/babylon/orbit2/Context.kt

```diff
@@ -16,11 +16,14 @@
 
 package com.babylon.orbit2
 
+import com.babylon.orbit2.idling.IdlingResource
+
 /**
  * Represents the current context in which an [Operator] is executing.
  *
  * @property state The current state captured at the point when the operator is executed
  * @property event The current event being processed
+ * @property idlingResource TODO Write description here
```

Comment:
🤔

Reply:
haha, well my TODO worked :) thoughts on how to describe it?